resources/views/locations/view.blade.php

```diff
@@ -155,16 +151,21 @@ class="table table-striped snipe-table"
           </div>
         @endif
 
-
       </div>
 
-
+		<div class="col-md-12">
+			<a href="{{ route('locations.edit', ['location' => $location->id]) }}" style="width: 60%;" class="btn btn-sm btn-primary pull-left">{{ trans('admin/locations/table.update') }} </a>
+		</div>
+        <div class="col-md-12" style="padding-top: 5px;">
+			<a href="{{ route('locations.printonlyassignedtolocation', ['location' => $location->id]) }}" style="width: 60%;" class="btn btn-sm btn-default pull-left">Print Only Assigned to Location</a>
```

Comment:
Can you use the translations instead of hard-coding english please?

Reply:
done